src/sqladmin-fetcher.ts

```diff
@@ -99,14 +99,34 @@ export class SQLAdminFetcher {
 
   async getEphemeralCertificate(
     {projectId, instanceId}: InstanceConnectionInfo,
-    publicKey: string
+    publicKey: string,
+    auth?: GoogleAuth
   ): Promise<SslCert> {
+
+    type RequestBody = {
+      public_key: string,
+      access_token?: string
+    }
```

Comment:
we should favour `interface` declarations over `type`, ref: https://google.github.io/styleguide/tsguide.html#interfaces-vs-type-aliases

I would also prefer to yank this to ~L28 so that it's declared right after the other interface declaration there 😊

Reply:
done!